### PR TITLE
Hardy/master/rhin 5870

### PR DESCRIPTION
**Pull request by hardy12994**
✔︎ I've tested in all browsers (IE11, Firefox, Safari, Chrome, and iOS)

### What should this PR do?
https://rhinogram.atlassian.net/browse/RHIN-5870
Changing selector view and functionality

### What is the acceptance criteria?
1. List

[If there are UI changes, include a before and after screenshot.]

### CODE REVIEW

File: src/less/forms.less [hunk 1]
```diff
@@ -602,6 +602,16 @@ input[type='checkbox']:focus {
   }
 }
 
+.form__group--wrapper {
+  position: relative;
+  min-height: 4rem;
```
Comment (Sonam-R) [inline src/less/forms.less:607]: can use @global-height

File: src/less/forms.less [hunk 2]
```diff
@@ -981,20 +991,49 @@ input[type='checkbox']:focus {
 .rhinoselect {
   position: relative;
   vertical-align: middle;
+  min-height: 4rem;
```
Comment (Sonam-R) [inline src/less/forms.less:994]: can use @global-height

File: src/less/forms.less [hunk 3]
```diff
@@ -981,20 +991,49 @@ input[type='checkbox']:focus {
...
 
+.select__label {
+  top: 0;
+  left: 0;
+  position: absolute;
+  border: #ccc solid 1px;
+  padding: 8px;
+  width: 100%;
+  font-weight: normal;
+  background-color: #fcfcfc;
```
Comment (Sonam-R) [inline src/less/forms.less:1007]: use consistent color as used in other places for background and border(use variable).

File: src/less/forms.less [hunk 4]
```diff
@@ -981,20 +991,49 @@ input[type='checkbox']:focus {
...
 
 .rhinoselect__select {
   cursor: pointer;
-  overflow: hidden;
+  // overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  opacity: 0;
+  height: 4rem;
+  min-height: 4rem;
```
Comment (Sonam-R) [inline src/less/forms.less:1022]: use variable

File: src/less/forms.less [hunk 5]
```diff
@@ -981,20 +991,49 @@ input[type='checkbox']:focus {
...
 //
 
 .rhinoselect__select {
   cursor: pointer;
-  overflow: hidden;
+  // overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  opacity: 0;
+  height: 4rem;
```
Comment (Sonam-R) [inline src/less/forms.less:1021]: do we need to use height here?

**Review by Sonam-R — APPROVED** [review]

**Review by marymhart — APPROVED** [review]

**Pull request merged**